cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -226,16 +223,20 @@ def chargeback_costs_default(resource_usage, db, provider):
     average_disk_io = resource_usage['average_disk_io']
 
     cpu_rate = query_rate(db, provider, 'cpu_usagemhz_rate_average', 'Default', 'Compute')
-    cpu_used_cost = average_cpu_used_in_mhz * float(cpu_rate) / 24
+    cpu_used_cost = version.pick({version.LOWEST: average_cpu_used_in_mhz * float(cpu_rate),
```

Comment:
i believe a `calculate_cost(usage, rate)` function should be even more readable in use